src/routes/comment.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+
+const router = express.Router();
+
+const { Comment } = require("../models/Comment");
+
+router.get("/:commentId", async (req, res) => {
+  try {
+    const commentId = req.params.commentId;
```

Comment:
```suggestion
    const { commentId } = req.params;
```
구조분해할당을 지향하면 좋을것같습니다.

Reply:
req.params.commentId는 엔드포인트인데 여기서 구조분해할당을 할 이유가 없어 보입니다

Reply:
params의 경우에도 여러게의 데이터를 객체로 전달할수있는것으로 알고있습니다.